src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs

```diff
@@ -120,7 +97,7 @@ public sealed record LedgerConfirmationOptions
     /// or equal to MaxCommitBatchSize.
     /// </summary>
     [ConfigurationKeyName("LargeBatchSizeToAddDelay")]
-    public long LargeBatchSizeToAddDelay { get; set; } = 100;
+    public long LargeBatchSizeToAddDelay { get; set; } = 500;
```

Comment:
for some reason we had different value in json file, i see no point in having different value in these 2 files.

Reply:
ditto